tests/Aspire.Dashboard.Tests/ConsoleLogsTests/LogEntriesTests.cs

```diff
@@ -31,4 +31,44 @@ public void InsertSorted_OutOfOrderWithSameTimestamp_ReturnInOrder()
             l => Assert.Equal("2", l.Content),
             l => Assert.Equal("3", l.Content));
     }
+
+    [Fact]
+    public void InsertSorted_TrimsToMaximumEntryCount_Ordered()
+    {
+        // Arrange
+        var logEntries = new LogEntries { MaximumEntryCount = 2, BaseLineNumber = 1 };
```

Comment:
Could you add a test with MaximumEntryCount = int.MaxValue? Just double check that items are still correctly added when there isn't really a limit.

Reply:
The first test covers this I think.